src/replication/log_consumer.cpp

```diff
@@ -371,7 +374,8 @@ namespace cubreplication
 
   void log_consumer::set_stop (void)
```

Comment:
I think we need to have this method blocking until all daemons are actually stopped/not running.
Rename method as "stop".